plugin/metrics/prometheus/metricsstore/reader.go

```diff
@@ -91,9 +91,6 @@ func NewMetricsReader(cfg config.Configuration, logger *zap.Logger, tracer trace
 	}
 
 	operationLabel := "operation"
-	if cfg.SupportSpanmetricsConnector {
-		operationLabel = "span_name"
```

Comment:
The logic doesn't look right here.

Reply:
What would be the appropriate option to be replaced there?

Reply:
The `operation` metric attribute was renamed to `span.name.`
Can be replaced with it?

Reply:
Yup, I'd expect: `operationLabel := "span_name"`

Reply:
Done!